rasa/core/policies/two_stage_fallback.py

```diff
@@ -76,20 +81,29 @@ def __init__(
         self.fallback_nlu_action_name = fallback_nlu_action_name
         self.deny_suggestion_intent_name = deny_suggestion_intent_name
 
+    @classmethod
+    def validate_against_domain(
+        cls, ensemble: "PolicyEnsemble", domain: Domain
```

Comment:
```suggestion
        cls, ensemble: Optional["PolicyEnsemble"], domain: Domain
```

Reply:
Then, shouldn't we also make `domain` also Optional?

Reply:
if it can be `None` then yes